Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -56,12 +56,10 @@ variable {𝕜 : Type*} [NontriviallyNormedField 𝕜]
   {E : Type*}
   [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type*} [TopologicalSpace H]
   (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] [ChartedSpace H M]
-  [SmoothManifoldWithCorners I M]
   -- declare a smooth manifold `M'` over the pair `(E', H')`.
```

Comment:
The comment does not really make sense any more. Maybe
```suggestion
  -- prerequisite typeclasses to say that `M'` is a smooth manifold over the pair `(E', H')`.
```
Same thing a few lines above and a few lines below